src/transformers/integrations.py

```diff
@@ -1537,6 +1547,68 @@ def on_save(self, args, state, control, **kwargs):
             self._clearml_task.update_output_model(artifact_path, iteration=state.global_step, auto_delete_file=False)
 
 
+class FlyteCallback(TrainerCallback):
+    """A [`TrainerCallback`] that sends the logs to [Flyte](https://flyte.org/).
+    NOTE: This callback only works within a Flyte task.
+
+    Args:
+        save_log_history (bool, optional, defaults to `True`):
+            When set to True, the training logs are saved as a Flyte Deck.
+
+        sync_checkpoints (bool, optional, defaults to `True`):
+            When set to True, checkpoints are synced with Flyte and can be used to resume training in the case of an
+            interruption.
+
+    Example:
+        ```python
+        # Note: This example skips over some setup steps for brevity.
+        from flytekit import current_context, task
+
+
+        @task
+        def train_hf_transformer():
+            cp = current_context().checkpoint
+            trainer = Trainer(..., callbacks=[FlyteCallback()])
+            output = trainer.train(resume_from_checkpoint=cp.restore())
+        ```
```

Comment:
Can be unindented of one level, and should have an additional line between Example and the three backticks.

Reply:
Good catch!